Travis build failing.

The Travis build is failing. The following error is being shown:

- FAILURE: Build failed with an exception.
 What went wrong:
A problem occurred configuring project ':app'.
 Could not resolve all dependencies for configuration ':app:_debugApkCopy'.
 Could not find com.github.deano2390:MaterialShowcaseView:1.1.0.
 Required by:
 project :app

**Would you like to work on the issue?**
<!-- Please let us know if you can work on it or the issue should be assigned to someone else. -->
- [X] Yes
- [ ] No

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -111,7 +111,7 @@ dependencies {
     compile 'com.google.android.exoplayer:exoplayer:r1.5.7'
 
     //utils
-    compile 'com.github.deano2390:MaterialShowcaseView:1.1.0'
+    compile 'com.github.deano2390:MaterialShowcaseView:1.2.0@aar'
     compile "com.jakewharton:butterknife:$rootProject.butterknifeVersion"
     annotationProcessor "com.jakewharton:butterknife-compiler:$rootProject.butterknifeVersion"
     compile 'com.github.bumptech.glide:glide:3.7.0'

